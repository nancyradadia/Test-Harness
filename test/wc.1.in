Hello
This is Nancy
This is a test file