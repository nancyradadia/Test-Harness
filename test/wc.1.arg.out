3 9 39	test/wc.1.in